x/dex/keeper/msg_server.go

```diff
@@ -23,12 +23,13 @@ var _ types.MsgServer = msgServer{}
 func (k msgServer) Deposit(goCtx context.Context, msg *types.MsgDeposit) (*types.MsgDepositResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	token0, token1, createrAddr, amount0, amount1, err := k.DepositVerification(goCtx, *msg)
+	token0, token1, autoswap, createrAddr, amount0, amount1, err := k.DepositVerification(goCtx, *msg)
```

Comment:
why does this need to be returned?

Reply:
This was from when autoswap was a param in `DepositCore`, but changing it since it's unnecessary